cmd/buildah/manifest.go

```diff
@@ -49,6 +50,11 @@ type manifestInspectOpts = struct {
 }
 
 func init() {
+	containerConfig, err := config.Default()
+	if err != nil {
+		logrus.Errorf(err.Error())
```

Comment:
Always use pass a format specifier as the first argument for functions like `Errorf()`, so that we don't have to track down surprises later.  I am concerned about parsing this again in this `init()`, when we're doing the same thing in both `from.go` and `main.go`.